Corona-Warn-App/src/main/java/de/rki/coronawarnapp/service/submission/SubmissionService.kt

```diff
@@ -75,6 +76,11 @@ object SubmissionService {
         return !(potentialGUID.isEmpty() || potentialGUID.length > SubmissionConstants.MAX_GUID_LENGTH)
     }
 
+    private fun matchGUIDPattern(scanResult: String): Boolean {
+        val pattern = Regex(SubmissionConstants.qrCodeRegex)
```

Comment:
Don't recompile the regex each time, put `Pattern.compile(....)` into a constant and re-use it. Regex pattern compilation can be costly.

Reply:
it's been done.  

note: not a const variable as "Pattern" is not Primitive data type